src/resolver/Mutation/update.js

```diff
@@ -1,6 +1,8 @@
 export default (entityData = []) => (_, params) => {
     const parsedId = parseInt(params.id, 10); // FIXME fails for non-integer ids
-    const indexOfEntity = entityData.findIndex(e => e.id === parsedId);
+    const indexOfEntity = entityData.findIndex(
+        e => parseInt(e.id, 10) === parsedId
```

Comment:
But Ids aren't always integers...

Reply:
Yes, but currently it's not working for non integers ids anyway

Reply:
Gotcha